internal/staging/primitives/uint256/uint256_test.go

```diff
@@ -0,0 +1,4176 @@
+		n:    "40000",
+		fmt:  "%#6o",
+		want: "01000000",
+	}, {
+		name: "decimal min width 5 w/ val > 5 digits and base prefix left pad",
+		n:    "186a0",
+		fmt:  "%#5d",
+		want: "100000",
+	}, {
+		name: "hex min width 12 w/ val > 12 digits and base prefix left pad",
+		n:    "1000000000000",
+		fmt:  "%#12x",
+		want: "0x1000000000000",
+	}, {
+		name: "binary min width 8 w/ val < 8 bits and base prefix zero pad",
+		n:    "a",
+		fmt:  "%#08b",
+		want: "0b001010",
+	}, {
+		name: "octal min width 6 w/ val < 6 digits and base prefix zero pad",
+		n:    "4551",
+		fmt:  "%#06o",
+		want: "042521",
+	}, {
+		name: "decimal min width 5 w/ val < 5 digits and base prefix zero pad",
+		n:    "270f",
+		fmt:  "%#05d",
+		want: "09999",
+	}, {
+		name: "hex min width 12 w/ val < 12 digits and base prefix zero pad",
+		n:    "abcdef",
+		fmt:  "%#012x",
+		want: "0x0000abcdef",
+	}, {
+		// ---------------------------------------------------------------------
+		// Mixed min width and precision with and without base prefix.
+		// ---------------------------------------------------------------------
+
+		name: "binary min width 8 min 6 bits w/ val 4 bits",
+		n:    "9",
+		fmt:  "%8.6b",
+		want: "  001001",
+	}, {
+		name: "octal min width 10 min 3 digits w/ val 2 digits and '0o' prefix",
+		n:    "d",
+		fmt:  "%10.3O",
+		want: "     0o015",
+	}, {
+		name: "decimal min width 12 min 7 digits w/ val 5 digits",
+		n:    "c34f",
+		fmt:  "%12.7d",
+		want: "     0049999",
+	}, {
+		name: "hex min width 32 min 16 digits w/ val 9 digits and base prefix",
+		n:    "89abcdef0",
+		fmt:  "%#32.16x",
+		want: "              0x000000089abcdef0",
+	}, {
+		// ---------------------------------------------------------------------
+		// Zero digits via precision with value == 0.
+		// ---------------------------------------------------------------------
+
+		name: "binary min 0 implied digits w/ val == 0",
+		n:    "0",
+		fmt:  "%.b",
+		want: "",
+	}, {
+		name: "octal min 0 digits w/ val == 0",
+		n:    "0",
+		fmt:  "%.0o",
+		want: "",
+	}, {
+		name: "decimal min 0 digits w/ val == 0 zero pad",
+		n:    "0",
+		fmt:  "%0.0d",
+		want: "",
+	}, {
+		name: "hex min 0 digits w/ val == 0 and base prefix",
+		n:    "0",
+		fmt:  "%#.0x",
+		want: "",
+	}, {
+		// ---------------------------------------------------------------------
+		// Misc.
+		// ---------------------------------------------------------------------
+
+		name: "unsupported format verb",
+		n:    "1000",
+		fmt:  "%f",
+		want: "%!f(Uint256=4096)",
+	}}
+
+	for _, test := range tests {
+		// Parse test hex.
+		n := hexToUint256(test.n)
+
+		got := fmt.Sprintf(test.fmt, n)
+		if got != test.want {
+			t.Errorf("%q: unexpected result -- got: %s, want: %s", test.name,
+				got, test.want)
+		}
+	}
+}
+
+// TestUint256ToBigRandom ensures that converting uint256s created from random
+// values to big ints works as expected.
+func TestUint256ToBigRandom(t *testing.T) {
```

Comment:
Noticed the `t.Parallel()` was added to all tests except for this one.

Reply:
Meh.  Missed that one it seems.  Added.